src/components/tooltip/bl-tooltip.ts

```diff
@@ -44,6 +45,39 @@ export default class BlTooltip extends LitElement {
 
   @state() private _visible = false;
 
+  /**
+   * Target elements state
+   */
+  @state() _target: string | Element;
+
+  /**
+   * Sets the target element of the popover to align and trigger.
+   * It can be a string id of the target element or can be a direct Element reference of it.
+   */
+  @property()
+  get target(): string | Element {
+    return this._target;
+  }
+
+  set target(value: string | Element) {
+    if (typeof value === 'string') {
+      this._target = document.getElementById(value) as Element;
+    } else if (value instanceof Element) {
+      this._target = value;
+    } else {
+      console.warn(
+        'BlTooltip target only accepts an Element instance or a string id of a DOM element.'
+      );
+    }
+
+    if (this._target instanceof Element) {
+      this._target.addEventListener('focus', this.show.bind(this));
+      this._target.addEventListener('blur', this.hide.bind(this));
```

Comment:
Listening to focus and blur events would be problemmatic on some elements like input. We will not want to show a tooltip when user focuses to an input by clicking it. I think we need to re-consider this behaviour.